### PR TITLE
cleanup(Styles): Cleanup Styles

### PR DESCRIPTION
**Pull request by TheGreatAxios**

### CODE REVIEW

File: src/content/docs/quick-start/building-for-skale.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+no monetary value or available market.
+
+### How to Take Payments
+
+It is quite common for dApps to use various mechanisms to represent value on chains.
+It is common to see many dApps use the native gas token of a chain -- Ether for example --
+to allow for payment. With sFUEL holding no monetary value, it means that alternative forms of value must be used.
+The most common way is to use ERC-20 tokens to represnet value.
+
+> #### Developer Tip
```
Comment (manuelbarbas) [inline src/content/docs/quick-start/building-for-skale.mdx:31]: Shouldn't be used instead the same pattern we're using in other developer notes?

:::tip[Developer Note]
:::

File: src/content/docs/quick-start/building-for-skale.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+When building on SKALE, you will notice that by default you get to cut out at least one step.
+Additionally, in situations where you are "covering the fees" you will not have to worry about variable
+costs since the gas is always free.
+
+<Tabs>
+	<TabItem label="Non-Custodial">
+		<Steps>
+			1. User downloads a wallet
+			2. sFUEL Distribution occurs -- _automatically in the background_
+			3. User begins interacting with dApp immediatley
```
Comment (manuelbarbas) [inline src/content/docs/quick-start/building-for-skale.mdx:78]: Would use the same sentence that's being used for custodial and AA: "User uses application"

File: src/content/docs/quick-start/building-for-skale.mdx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+			2. User visits CEX or Fiat onramp
+			3. *Optional* - User bridges to new network
+			4. User pays fees and clicks and manually approves every transaction
+		</Steps>
+	</TabItem>
+	<TabItem label="Custodial">
+		<Steps>
+			1. User logs in to application
+			2. User uses application
+			3. dApp pays the gas fees
```
Comment (manuelbarbas) [inline src/content/docs/quick-start/building-for-skale.mdx:56]: Question: would it make sense to add another step before the last one saying something like `dApp needs to carefully control  what goes on-chain due to gas costs`

File: src/content/docs/quick-start/building-for-skale.mdx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+	<TabItem label="Account Abstraction">
+		<Steps>
+			1. User logs in to application
+			2. An MPC, custodial, or background wallet is created 
+			3. User uses application
+			4. dApp pays the gas fees via paymaster
+		</Steps>
+	</TabItem>
+</Tabs>
+
```
Comment (manuelbarbas) [inline src/content/docs/quick-start/building-for-skale.mdx:68]: For quicker and clearer comprehension of the diferences I would add a sub-title like:

##### (On) SKALE Chains

File: src/content/docs/quick-start/building-for-skale.mdx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+> #### Developer Tip
+>
+> See the [community guide](https://hackmd.io/@TheGreatAxios/SJWksGU2i) on how to use
+> ERC-20 instead of msg.value
+
+### Onboarding & sFUEL Distribution
+
+Onboarding users -- regardless of whether you are using non-custodial or custodial wallets -- is
+unique when it comes to SKALE.
+
```
Comment (manuelbarbas) [inline src/content/docs/quick-start/building-for-skale.mdx:40]: For quicker and clearer comprehension of the diferences I would add a sub-title like:

##### (On) Other Chains

File: src/content/docs/quick-start/building-for-skale.mdx (hunk 1 above)
Comment (TheGreatAxios) [reply src/content/docs/quick-start/building-for-skale.mdx]: Resolved by [af389b8](https://github.com/skalenetwork/docs.skale.space/pull/108/commits/af389b83a86d089e6066f96cac399adcd982686c)

File: src/content/docs/quick-start/building-for-skale.mdx (hunk 2 above)
Comment (TheGreatAxios) [reply src/content/docs/quick-start/building-for-skale.mdx]: Resolved by [aba73a7](https://github.com/skalenetwork/docs.skale.space/pull/108/commits/aba73a721fc99d253faab84b3d9038b92696ba53)

File: src/content/docs/quick-start/building-for-skale.mdx (hunk 4 above)
Comment (TheGreatAxios) [reply src/content/docs/quick-start/building-for-skale.mdx:68]: Resolved by [77aa5a5](https://github.com/skalenetwork/docs.skale.space/pull/108/commits/77aa5a5d4f739eb58faa6810cafb6e63278234aa)

File: src/content/docs/quick-start/building-for-skale.mdx [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+It is common to see many dApps use the native gas token of a chain -- Ether for example --
+to allow for payment. With sFUEL holding no monetary value, it means that alternative forms of value must be used.
+The most common way is to use ERC-20 tokens to represnet value.
+
+> #### Developer Tip
+>
+> See the [community guide](https://hackmd.io/@TheGreatAxios/SJWksGU2i) on how to use
+> ERC-20 instead of msg.value
+
+### Onboarding & sFUEL Distribution
```
[the remark this answers is not in the document]
Comment (TheGreatAxios) [reply src/content/docs/quick-start/building-for-skale.mdx:36]: Will add under SKALE Onboarding once updated in **App Developers**

**Review by manuelbarbas — APPROVED** [review]

**Pull request merged**